core/src/wallet/ripple/explorers/NodeRippleLikeBlockchainExplorer.cpp

```diff
@@ -154,18 +154,17 @@ namespace ledger {
                             !json["result"].IsObject()) {
                             throw make_exception(api::ErrorCode::HTTP_ERROR, "Failed to broadcast transaction, no (or malformed) field \"result\" in response");
                         }
-                        //Is there an tx_json field ?
+
                         auto resultObj = json["result"].GetObject();
-                        if (!resultObj.HasMember("tx_json") || !resultObj["tx_json"].IsObject()) {
-                            throw make_exception(api::ErrorCode::HTTP_ERROR, "Failed to broadcast transaction, no (or malformed) field \"tx_json\" in response");
-                        }
 
-                        //Is there an hash field ?
-                        auto jsonObj = resultObj["tx_json"].GetObject();
-                        if (!jsonObj.HasMember("hash") || !jsonObj["hash"].IsString()) {
-                            throw make_exception(api::ErrorCode::HTTP_ERROR, "Failed to broadcast transaction, no (or malformed) field \"hash\" in response");
+                        if (resultObj.HasMember("engine_result") && resultObj["engine_result"] == "tesSUCCESS") {
```

Comment:
Do you know what happens if `resultObj["engine_result"]` is not there? Can it be compared to a string as you do? I guess yes but I’d like to be sure :D

Reply:
If `resultObj["engine_result"]` is not there, I guess the second condition `resultObj["engine_result"] == "tesSUCCESS"` will be skipped. I could be wrong, but that's the behaviour in 🐍.

Reply:
Oh yeah I had not seen the first part of the test. LGTM.